eng/TestProperties.props

```diff
@@ -12,4 +12,9 @@
     <WcfScenarioTestCommonProj>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)\..\src\System.Private.ServiceModel\tests\Common\Scenarios\ScenarioTests.Common.Tests.csproj'))</WcfScenarioTestCommonProj>
     <WcfInfrastructureCommonProj>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)\..\src\System.Private.ServiceModel\tests\Common\Infrastructure\Infrastructure.Common.csproj'))</WcfInfrastructureCommonProj>
   </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
this will need to be in a separate .proj file and you'll need to make sure you execute msbuild on that proj file. You'll also need to make sure you're including the workitems.